stackdriver_debugger.c

```diff
@@ -496,3 +522,14 @@ PHP_INI_MH(OnUpdate_stackdriver_debugger_max_time)
     }
     return SUCCESS;
 }
+
+/**
+ * Callback for when the user changes the max memory php.ini setting.
+ */
+PHP_INI_MH(OnUpdate_stackdriver_debugger_max_memory)
+{
+    if (new_value != NULL) {
+        STACKDRIVER_DEBUGGER_G(max_memory) = 1000000 * ZEND_STRTOL(ZSTR_VAL(new_value), NULL, 0);
```

Comment:
Not 1048576?

Reply:
You're right, other PHP conversions from use 1024 for 1k, so we'll change to 1048576 (1024*1024)